### ISSUE TITLE
Stable Root Element (Glimmer 2)

### ISSUE THREAD

**Issue by 256dpi** [cibernox/ember-basic-dropdown#193]
This addon seems to be almost compatible with Glimmer 2. However the wormholed dropdown sometimes cauces an error in my app. I guess this is related to the need for a stable root element as mentioned in the latest [ember-wormhole release](https://github.com/yapplabs/ember-wormhole/releases/tag/0.5.1).

This:

```hbs
{{#ember-wormhole to=to renderInPlace=renderInPlace}}
  {{#if overlay}}
    <!-- ... -->
  {{/if}}
{{/ember-wormhole}}  
```

Needs to be changed to something like:

```hbs
{{#ember-wormhole to=to renderInPlace=renderInPlace}}
  <div class="something">
    {{#if overlay}}
      <!-- ... -->
    {{/if}}
  </div>
{{/ember-wormhole}}  
```

Would that be feasible?

**Comment by cibernox** [comment]
I'm hesitant to add that div for a bug that should be fixed soon (so they say). 
I'd rather add a comment for glimmer2 users in the readme that due to that bug you need to add 
`<div>your content here</div>` div until that bug is solved and link to it the open issue in ember-wormhole.

**Comment by cibernox** [comment]
@256dpi You don't have to worry about EPS, since it already adds a stable root. This is only something to be aware when using raw `ember-basic-dropdown`.

**Comment by 256dpi** [comment]
👍  Thanks for clarifying!

**Comment by cibernox** [comment]
@256dpi Actually I was wrong. EPS does not add  that div. I wonder how is this working for me. 

If anything I'll add that div to EPS itself. I'll check why it works.

**Comment by 256dpi** [comment]
In the same moment the error hit me again in my app:

```
vendor.js?v=0.7.4:42 Uncaught Error: ember-wormhole failed to render into '#ember-basic-dropdown-wormhole' because the element is not in the DOM
    at i._appendToDestination (vendor.js?v=0.7.4:42)
    at vendor.js?v=0.7.4:42
    at invoke (vendor.js?v=0.7.4:5)
    at s.flush (vendor.js?v=0.7.4:5)
    at u.flush (vendor.js?v=0.7.4:5)
    at d.end (vendor.js?v=0.7.4:5)
    at vendor.js?v=0.7.4:5
```

**Comment by cibernox** [comment]
Actully, I was wrong in everything. It's EPS the one who adds the wrapped div. The only issue is when the overlay is active, which I don't use much and I haven't noticed. I'll investigate this.

**Comment by 256dpi** [comment]
FYI: I'm not using the overlay option.

**Comment by cibernox** [comment]
@256dpi interestingly, I can't reproduce the error in this repo. Can you come up with a reproduction? Perhaps it has already been fixed in the lates 2.10.X?

**Comment by 256dpi** [comment]
Actually my issue is more related to that: https://github.com/cibernox/ember-power-select/issues/145.
I'm not using the generated .html file to serve the app, therefore the div does not exist.
